.jshintignore

```diff
@@ -1,7 +1,7 @@
 nuve/nuveClient/build
 nuve/nuveClient/dist
-erizo_controller/erizoClient
-spine/erizofc.js
+erizo_controller/erizoClient/**/*.js
+spine/**/*.js
```

Comment:
It helps when you have an editor configured with both eslint and jshint.
The idea is to create a gulp task or similar in spine to lint it, will do in a future PR.